src/crypto/crypto.h

```diff
@@ -68,71 +67,77 @@ namespace crypto
 {
 
 #pragma pack(push, 1)
-POD_CLASS ec_point
+struct ec_point
 {
 	char data[32];
 };
 
-POD_CLASS ec_scalar
+struct ec_scalar
 {
 	char data[32];
 };
 
-POD_CLASS public_key : ec_point
+struct public_key : ec_point
 {
 	friend class crypto_ops;
 };
 
-POD_CLASS scalar_16
+struct scalar_16
 {
 	uint8_t data[16];
 };
 
 using secret_key_16 = tools::scrubbed<scalar_16>;
 using secret_key = tools::scrubbed<ec_scalar>;
 
-POD_CLASS public_keyV
+struct public_keyV
 {
 	std::vector<public_key> keys;
 	int rows;
 };
 
-POD_CLASS secret_keyV
+struct secret_keyV
 {
 	std::vector<secret_key> keys;
 	int rows;
 };
 
-POD_CLASS public_keyM
+struct public_keyM
 {
 	int cols;
 	int rows;
 	std::vector<secret_keyV> column_vectors;
 };
 
-POD_CLASS key_derivation : ec_point
+struct key_derivation : ec_point
 {
 	friend class crypto_ops;
 };
 
-POD_CLASS key_image : ec_point
+struct key_image : ec_point
 {
 	friend class crypto_ops;
 };
 
-POD_CLASS signature
+struct signature
 {
 	ec_scalar c, r;
 	friend class crypto_ops;
 };
+
+struct uniform_payment_id
+{
+	uint64_t zero = 0;
```

Comment:
could you please add a doc string which describes for what this variable is used and what is the meaning. I see later that it is set to `-1` if there is no payment id and also compared with `0` (what is the meaning if the zero id is `0`

Reply:
`zero == 0` means the structure contains valid data. I will add that to the docs.